src/check-local-deployer.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace Deployer;
+
+desc('Check currently used deployer path');
+task('s24:check-local-deployer', function () {
+
+    $scriptPath = get_included_files()[0];
+
+    if (strpos($scriptPath, __DIR__.'/vendor/deployer/deployer/bin/dep') !== true) {
+        throw new \RuntimeException("Pleaae run using local deployer with ./vendor/bin/dep");
```

Comment:
Perfect! I'd capitalise "Deployer" in the exception so its clear you're talking about the Deployer software

Reply:
Updated